### PR TITLE
Traces IA

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Improves and streamlines Traces IA. Clearer sidebar descriptions, more intuitive organization.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/api/tracing.md [hunk 1]
```diff
@@ -396,7 +396,7 @@ Select the documentation link for your deployment:
 
 ## Traces API
 
-The Traces API allows you to browse traces collected in the system. You can execute queries to find traces matching provided search criteria as well as gather detailed information about individual traces and spans. For more information see [View and investigate traces](/docs/apm/traces/Working-with-Tracing-data/View-and-investigate-traces).
+The Traces API allows you to browse traces collected in the system. You can execute queries to find traces matching provided search criteria as well as gather detailed information about individual traces and spans. For more information see [View and investigate traces](/docs/apm/traces/View-and-investigate-traces).
```
Comment (kimsauce) [inline docs/api/tracing.md:399]:
```suggestion
The Traces API allows you to browse traces collected in the system. You can execute queries to find traces matching provided search criteria as well as gather detailed information about individual traces and spans. For more information, see [View and investigate traces](/docs/apm/traces/View-and-investigate-traces).
```

File: docs/api/tracing.md [hunk 2]
```diff
@@ -215,7 +215,7 @@ Select the documentation link for your deployment:
 
 ## Span Analytics API
 
-The Span Analytics API allows you to browse spans collected in the system. You can execute queries to find individual spans matching provided search criteria as well as run aggregated span queries and retrieve their results. For more information see [Spans](/docs/apm/traces/working-with-tracing-data/spans.md).
+The Span Analytics API allows you to browse spans collected in the system. You can execute queries to find individual spans matching provided search criteria as well as run aggregated span queries and retrieve their results. For more information see [Spans](/docs/apm/traces/spans.md).
```
Comment (kimsauce) [inline docs/api/tracing.md:218]:
```suggestion
The Span Analytics API allows you to browse spans collected in the system. You can execute queries to find individual spans matching provided search criteria as well as run aggregated span queries and retrieve their results. For more information, see [Spans](/docs/apm/traces/spans.md).
```

File: docs/apm/traces/spans.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
 ---
 id: spans
 title: Analyze data on Span level
-sidebar_label: Analyze data on Span level
+sidebar_label: Spans
```
Comment (kimsauce) [inline docs/apm/traces/spans.md:4]:
```suggestion
sidebar_label: Analyze data on Span level
```

File: docs/apm/traces/tracing-dashboards.md [hunk 4]
```diff
@@ -1,14 +1,15 @@
 ---
 id: tracing-dashboards
 title: Application Service APM/Tracing Dashboards
+sidebar_label: Tracing/APM Dashboards
```
Comment (kimsauce) [inline docs/apm/traces/tracing-dashboards.md:4]:
```suggestion
```

**Review by pbrzoska-sumo — CHANGES_REQUESTED** [review]
1. Top level menu link should go to /docs/apm/ not to /docs/apm/traces/
2. set up traces collection should be first, then instrumenting. The latter one assumes the first one is done, so order is important
3. Set up traces collection should be called out in the menu, possibly on the same level as OT app instrumentation, similarly as we define that as sections in Transaction Tracing Setup page
4. Please move dashboards section above span analytics one
5. http://localhost:3000/docs/apm/real-user-monitoring/configure-data-collection/#step-3-search-traces-from-the-browser should be a separate section. We will ad more content there.

File: docusaurus.config.js [hunk 5]
```diff
@@ -307,13 +307,13 @@ module.exports = {
...
-                label: 'Traces',
-                to: '/docs/apm/traces',
+                label: 'Traces/APM/RUM',
+                to: '/docs/apm',
                 activeBaseRegex: '^/docs/apm/.*',
               },
               {
                 label: 'Alerts/Monitors',
-                to: '/docs/alerts',
+                to: '/docs/alerts/.*',
```
Comment (kimsauce) [inline docusaurus.config.js:316]:
```suggestion
                to: '/docs/alerts',
```

**Review by pbrzoska-sumo — APPROVED** [review]

**Review by MariePMcGarry — APPROVED** [review]

**Pull request merged**